### DISCUSSION TITLE
Single product images not showing

### DISCUSSION THREAD

**Discussion by sihamsa** [breezefront/community#302] [1. Help]
hey i have a problem in my single product page in Wordpress I can only see the main  images but the other product images can't see them when I click on the quick view 
![Capture d’écran 2024-07-14 120341](https://github.com/user-attachments/assets/83890bd4-3f66-469c-b442-1b3b0ea4d4d3)
![Capture d’écran 2024-07-14 120425](https://github.com/user-attachments/assets/39d44242-8954-4b01-81d2-1ad817ca69b0)

**Comment by vovayatsyuk** [comment]
Please provide a URL to this page.

**Reply by sihamsa** [reply]
The picture in the left
https://kasa.ma/product/salon-tokyo/

**Reply by vovayatsyuk** [reply]
Hmm, I'm not quite sure I understand your request. Your store uses WordPress. It's not a Breezefront. You need to contact your theme developers.
